platform/solr/platform-solr-server-standalone/src/main/resources/solr/conf/solrconfig-inmemory.xml

```diff
@@ -1159,9 +1159,10 @@
             <str name="suggestAnalyzerFieldType">lowercase</str>
             <str name="field">title_txt</str>
             <str name="contextField">metacard-tags_txt</str>
-            <str name="weightField">ext.population_lng</str>
+            <str name="weightField">ext.gazetteer-sort-value_int</str>
+            <str name="sortField">ext.population_lng</str>
             <str name="payloadField">id_txt</str>
-            <str name="buildOnStartup">false</str>
+            <str name="buildOnStartup">true</str>
```

Comment:
This might cause problems in itests where Gazetteer data is present. The index build can take a long time, which will cause PaxExam to time out.